modules/mongodb/manifests/monitoring.pp

```diff
@@ -70,5 +70,6 @@
     check_command       => 'check_nrpe!check_mongodb!replset_state',
     service_description => 'mongod replset state',
     host_name           => $::fqdn,
+    notes_url           => monitoring_docs_url(howto-recreate-a-mongo-instance-in-aws),
```

Comment:
[Ditto](https://github.com/alphagov/govuk-puppet/pull/7304/files#r170865357).

Reply:
Ditto your ditto: https://github.com/alphagov/govuk-developer-docs/pull/735/files

Reply:
I've moved the file into the alerts folder

Reply:
It might be good to rename that file so that it's clear it's really about an alert. Also, I'd update the metadata to add to the "alerts" section. It's currently under "development VM" even though it's an AWS thing.

Reply:
Done here: https://github.com/alphagov/govuk-developer-docs/pull/738/files

Reply:
Should this drop the "howto-" now that the file has been renamed?

Reply:
Indeed, renamed now.